docs/source/conf.py

```diff
@@ -89,6 +90,20 @@
 # The full version, including alpha/beta/rc tags.
 release = cartopy.__version__
 
+# Sphinx gallery configuration
+sphinx_gallery_conf = {
+    'examples_dirs': ['../../lib/cartopy/examples'],
+    'filename_pattern': '^((?!sgskip).)*$',
+    'gallery_dirs': ['gallery'],
+    'doc_module': ('cartopy',),
+    'reference_url': {'matplotlib': 'http://matplotlib.org',
+                      'numpy': 'http://docs.scipy.org/doc/numpy/reference',
+                      'scipy': 'http://docs.scipy.org/doc/scipy/reference',
+                      'shapely': 'http://toblerity.org/shapely',
+                      'cartopy': None},
```

Comment:
These two lines should be removable at this point?

(the second line is the ```backreferences_dir``` one)

Reply:
you want to remove the `backreferences_dir` line? I'm not sure which you're referring to